crates/builder/src/lib.rs

```diff
@@ -0,0 +1,4 @@
+pub mod config;
+pub mod service;
+pub mod signer;
+pub mod tasks;
```

Comment:
Exposed `builder` as a library for integration tests. `main` now calls the `builder` like a library in the workspace.

Reply:
cool. should it be in a `bin/` folder?